src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Ensure that the `ColorSupplier` class is implemented with a `getRandomColor()` method that returns a random color from the `Color` enum. This is necessary for the `getRandomBall()` method to function correctly.